client/components/App.js

```diff
@@ -27,15 +29,14 @@ class App extends React.Component {
     this.getData();
   }
 
-  //  18.191.88.165
   getData() {
-    axios.get('http://localhost:3000/reviews')
+    axios.get(`/reviews/${Math.floor(Math.random() * 10000000)}`)
       .then((result) => this.setState({ allReviews: result.data }))
       .catch((err) => console.log(err, 'error'));
   }
 
   submitReview(input) {
-    axios.post('http://localhost:3000/review', { input })
+    axios.post('/reviews', input)
```

Comment:
did you need to remove localhost? if it works, it works

Reply:
yes